### PR TITLE
Version check

### PR DESCRIPTION
**Pull request by pooya-mohammadi**
Before pushing to pypi check whether pyproject's version and giit tag's version are the same!

### CODE REVIEW

**Comment by pooya-mohammadi** [comment]
@arxyzan jan, please check this

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -49,6 +51,10 @@ jobs:
             echo "[Error]: Version Miss Match!"
             exit 1
           fi    
+          if [[ ${{ env.GIT_TAG }} != "${{ env.init_git_version }}" ]]; then
+            echo "[Error]: Version Miss Match!"
```
Comment (arxyzan) [inline .github/workflows/pypi-release.yml:55]: mismatch* Pooya jan :)

**Comment by pooya-mohammadi** [comment]
Fixed typo at [80dd513](https://github.com/hezarai/hezar/pull/63/commits/80dd513eedb2b2dbc7e0fefc443078b9a968065d)

**Review by arxyzan — APPROVED** [review]

**Comment by pooya-mohammadi** [comment]
@arxyzan I also added missmatch for __init__ this will prevent inconsistent tags

**Pull request merged**